eng/WpfArcadeSdk/tools/SdkReferences.targets

```diff
@@ -202,10 +202,10 @@
                  Condition="Exists('$(PkgMicrosoft_Private_Winforms)\ref\$(TargetFramework)\%(MicrosoftPrivateWinFormsReference.Identity).dll')">
         <NuGetPackageId>Microsoft.Private.Winforms</NuGetPackageId>
       </Reference>
-      <Reference Include="$(PkgMicrosoft_Private_Winforms)\ref\netcoreapp5.0\%(MicrosoftPrivateWinFormsReference.Identity).dll"
+      <Reference Include="$(PkgMicrosoft_Private_Winforms)\ref\$(TargetFramework)\%(MicrosoftPrivateWinFormsReference.Identity).dll"
```

Comment:
You cannot use `$(TargetFramework)` here since it's used as a fallback for the other `Reference` above this one which uses `$(TargetFramework)`. The fallback is used when WPF increments the target framework version before WinForms. We can see it now with #5901 blocking #5908 and vice versa. This must be the target framework of the current Microsoft.Private.Winforms (net6.0).